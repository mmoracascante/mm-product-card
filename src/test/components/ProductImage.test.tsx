import React from "react"; import renderer from 'react-test-renderer'
import { ProductCard, ProductImage } from '../../components'
import { product2 } from '../data/products'



describe('Testeando componente <ProductImage />', () => {

    test('Debe mostrar la imagen', () => {
        const wrapper = renderer.create(
            <ProductCard product={product2}>

                {
                    () => (<ProductImage />)
                }
            </ProductCard>



        )
        expect(wrapper.toJSON()).toMatchSnapshot()


    })

    test('No debe mostrar la imagen', () => {
        const wrapper = renderer.create(
            <ProductImage img="https://hola.jpg" />



        )
        expect(wrapper.toJSON()).toMatchSnapshot()


    })
})