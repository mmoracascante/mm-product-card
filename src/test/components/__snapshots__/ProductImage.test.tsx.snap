// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testeando componente <ProductImage /> Debe mostrar la imagen 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`Testeando componente <ProductImage /> No debe mostrar la imagen 1`] = `
<img
  alt="Product"
  className="productImg undefined"
  src="https://hola.jpg"
/>
`;
